crates/ruff_python_semantic/src/definition.rs

```diff
@@ -148,6 +151,17 @@ impl<'a> Definition<'a> {
         )
     }
 
+    pub fn is_property(
+        &self,
+        extra_properties: &[QualifiedName],
+        semantic: &SemanticModel,
+    ) -> bool {
+        self.as_function_def()
+            .is_some_and(|StmtFunctionDef { decorator_list, .. }| {
+                is_property(decorator_list, extra_properties, semantic)
+            })
+    }
+
```

Comment:
I wonder if it's worth having this method in `definition` if it only ever is called from `pylint` rules. I would move it upwards next to the lint rule.

Reply:
I'd prefer to have it here, so that it can be easily reused by other rules that we add in the future